pipeline/gke-deploy-build-push/0.1/support/gke-deploy.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gke-deploy
+spec:
+  params:
```

Comment:
can we can use the existing [gke-deploy](https://github.com/tektoncd/catalog/blob/master/task/gke-deploy/0.1/gke-deploy.yaml) task over since that task is a more generic?